### PR TITLE
Update description in regionaltransit (RTA) spider

### PR DESCRIPTION
**Pull request by o-stovicek**
From issue #275 
The code around the description was a bit different from the other spider I edited, so hopefully this works.

### CODE REVIEW

File: documenters_aggregator/spiders/regionaltransit.py [hunk 1]
```diff
@@ -17,7 +17,12 @@ class RegionaltransitSpider(Spider):
     domain_root = 'http://www.rtachicago.org'
 
     def parse_iframe(self, response):
-        description = response.request.meta['description']
+        description = ("The RTA Board of Directors is a 16-member group of professionals"
+                      "governing the activities and initiatives of the RTA. The RTA is "
+                      "charged with financial oversight, funding, and regional transit "
+                       "planning for the region’s transit operators: the Chicago Transit
```
Comment (easherma) [inline documenters_aggregator/spiders/regionaltransit.py]: looks like you're just missing a closing quote at the end of this line!

File: tests/test_regionaltransit.py [hunk 2]
```diff
@@ -16,7 +16,12 @@ def test_name():
 
 
 def test_description():
-    assert parsed_items[0]['description'] == 'Test Description'
+    assert 'Test Description' == ("The RTA Board of Directors is a 16-member group of professionals"
+                        "governing the activities and initiatives of the RTA. The RTA is "
+                        "charged with financial oversight, funding, and regional transit "
+                        "planning for the region’s transit operators: the Chicago Transit
```
Comment (easherma) [inline tests/test_regionaltransit.py]: it looks like a closing quote is missing here as well. Also, Python is sensitive to [indentation](https://stackoverflow.com/questions/13884499/what-is-python-whitespace-and-how-does-it-work).  I'd recommend formatting these quotes to look more like the description in your spider, including the way the quotes are indented (which is essential) and the spacing (which just helps readability)

**Comment by o-stovicek** [comment]
Passing checks now, so it would be great if someone could take a look at this!

**Comment by bonfirefan** [comment]
@o-stovicek this looks good, merging it in! Thanks for the reviews, @easherma

**Pull request merged**